? Makefile
? Makefile.in
? acinclude.m4
? aclocal.m4
? admin
? arson-crashfix.diff
? config.h
? config.h.in
? config.log
? config.status
? configure
? configure.in
? configure.in.in
? libtool
? stamp-h
? stamp-h.in
? stamp-h1
? doc/Makefile
? doc/Makefile.in
? doc/arson/Makefile
? doc/arson/Makefile.in
? po/Makefile
? po/Makefile.in
? po/de.gmo
? po/es.gmo
? po/sv.gmo
? src/.deps
? src/.libs
? src/Makefile
? src/Makefile.am
? src/Makefile.in
? src/_burnerpage.cpp
? src/_burnerpage.h
? src/_burnerpage.moc
? src/_cdcopydlg.cpp
? src/_cdcopydlg.h
? src/_cdcopydlg.moc
? src/_cdinfoeditor.cpp
? src/_cdinfoeditor.h
? src/_cdinfoeditor.moc
? src/_configdlg.cpp
? src/_configdlg.h
? src/_configdlg.moc
? src/_devicepage.cpp
? src/_devicepage.h
? src/_devicepage.moc
? src/_isofs.cpp
? src/_isofs.h
? src/_isofs.moc
? src/_isopage.cpp
? src/_isopage.h
? src/_isopage.moc
? src/_listview.cpp
? src/_listview.h
? src/_listview.moc
? src/_lookupdlg.cpp
? src/_lookupdlg.h
? src/_lookupdlg.moc
? src/_optionpage.cpp
? src/_optionpage.h
? src/_optionpage.moc
? src/_programdlg.cpp
? src/_programdlg.h
? src/_programdlg.moc
? src/_programpage.cpp
? src/_programpage.h
? src/_programpage.moc
? src/_progressdlg.cpp
? src/_progressdlg.h
? src/_progressdlg.moc
? src/_ripperpage.cpp
? src/_ripperpage.h
? src/_ripperpage.moc
? src/_textviewer.cpp
? src/_textviewer.h
? src/_textviewer.moc
? src/_waitdlg.cpp
? src/_waitdlg.h
? src/_waitdlg.moc
? src/arson
? src/arson_meta_unload.cpp
? src/burnerpage.moc.cpp
? src/cdripper.moc.cpp
? src/configdlg.moc.cpp
? src/datadoc.moc.cpp
? src/devicepage.moc.cpp
? src/docwidget.moc.cpp
? src/infprogbar.moc.cpp
? src/isofs.moc.cpp
? src/isopage.moc.cpp
? src/listwnd.moc.cpp
? src/logwnd.moc.cpp
? src/mainwnd.moc.cpp
? src/optionpage.moc.cpp
? src/process.moc.cpp
? src/programdlg.moc.cpp
? src/programpage.moc.cpp
? src/progressdlg.moc.cpp
? src/ripperpage.moc.cpp
? src/socket.moc.cpp
? src/pics/Makefile
? src/pics/Makefile.in
Index: src/docwidget.cpp
===================================================================
RCS file: /cvsroot/arson/arson/src/docwidget.cpp,v
retrieving revision 1.9
diff -u -3 -p -r1.9 docwidget.cpp
--- src/docwidget.cpp	26 Oct 2002 19:06:47 -0000	1.9
+++ src/docwidget.cpp	21 Nov 2002 17:46:07 -0000
@@ -516,8 +516,10 @@ ArsonLvPos::ArsonLvPos (ArsonListDoc *pd
 ArsonListDoc::ArsonListDoc (QWidget *parent, const char *name)
 	: ArsonDocWidget(parent, name),
 	m_renameCol(-1),
-	m_editableList(true)
+	m_editableList(true),
+	m_pList(NULL)
 {
+	//	Nothing...
 }
 
 ArsonListDoc::~ArsonListDoc (void)
